tests/Unit/LoanServiceTest.php

```diff
@@ -56,8 +56,8 @@ public function testServiceCanCreateLoanOfForACustomer()
         ]);
         $this->assertDatabaseHas('scheduled_repayments', [
             'loan_id' => $loan->id,
-            'amount' => 1666,
-            'outstanding_amount' => 1666,
+            'amount' => 1667,
+            'outstanding_amount' => 1667,
```

Comment:
This had to be changed to `1667` as the previous test cases had 1666, 1666 and 1667 as the expected amounts, which sum up to be 4999 (not equal to the loan amount, 5000)